scripts/people_styling.js

```diff
@@ -0,0 +1,64 @@
+
+
+function displayImage(img) {
+	let div = document.createElement("div");
+	let copy = document.createElement("img");
+	copy.src = img.src;
+	div.id = "popup";
+	div.appendChild(copy);
+	document.body.insertBefore(div, document.body.firstChild);
+	img.style.display = "none"; // hide original image
+	div.addEventListener("click", () => {
+		document.body.removeChild(div)
```

Comment:
```suggestion
		document.body.removeChild(div);
```
not required, but for uniformity